src/pkg/project/dao/dao.go

```diff
@@ -70,7 +70,7 @@ func (d *dao) Create(ctx context.Context, project *models.Project) (int64, error
 			return orm.WrapConflictError(err, "The project named %s already exists", project.Name)
 		}
 
-		member := &Member{
+		member := &models.Member{
```

Comment:
There is another `Member` definition under package `pkg/member`, seems they are duplicated?

Reply:
will raise another PR to handle this.